great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -2898,27 +2899,23 @@ def _test_instantiation_of_misc_class_from_yaml_config(
 
         return instantiated_class, usage_stats_event_payload
 
-    def _instantiate_datasource_from_config_and_update_project_config(
+    def _instantiate_datasource_from_config_and_update_project_config(  # type: ignore
```

Comment:
any specific warning we're ignoring here? I'd like to use the `[specific-warning]` syntax as much as possible

Reply:
Good catch, missed this one. Fixing.

Reply:
Fixed!